comps/dataprep/utils.py

```diff
@@ -276,7 +276,8 @@ def load_json(json_path):
     """Load and process json file."""
     with open(json_path, "r") as file:
         data = json.load(file)
-    return json.dumps(data)
+    content_list = [json.dumps(item) for item in data]
```

Comment:
what is the fields/format in json/jsonl file?
{"text": "xxx"}?
{"content": "xxx"}

I don't see how to extract the text from json/jsonl as chunk to ingest

Reply:
considering that most of json/jsonl data exhibit their own data format in each line, e.g., {"text":xxx, "docid":xxx, "url":xxx}. We can directly output each line of data as a single data instance.